localization/english/spark_council_l_english.yml

```diff
@@ -4,7 +4,7 @@
 	aeluran_advisor_desc:0 "The Aeluran Advisor is a powerful and influential figure in the court. They are responsible for the recruitment and training of the Magi, the advisor's personal army of sorcerers."
 
 	game_concept_aeluran_advisor:0 "Aeluran Advisor"
- 	game_concept_aeluran_advisor_desc:0 "The $game_concept_aeluran_advisor$ is the [aeluran_advisor|E], second in command of your lands, in charge of managing your Magi core."
+ 	game_concept_aeluran_advisor_desc:0 "The $game_concept_aeluran_advisor$ is the [aeluran_advisor|E], second in command of your lands, in charge of managing the core of you Magi."
```

Comment:
I was intending to use core as like military regiment type name but I realize now it would be corp like "Marine Corps"?

maybe just "managing your Magi forces"?